sdk/python/foundation-models/system/docs/component_docs/nlp_finetune/finetune_component_parameters.md

```diff
@@ -0,0 +1,210 @@
+# Finetune Pipeline Component
+This component enables finetuning of pretrained models on custom or pre-available datasets. The component supports Deepspeed for performance enhancement. The component supports parameter efficient finetuning with techniques like LoRA.
+
+At the time of writing, following tasks are supported through finetuning components:
+1. Text Generation
+2. Text Classification
+3. Named Entity Recognition/Token Classification
+4. Question Answering
+5. Summarization
+6. Translation
+
+Respective components can be found in the azureml registry:
+- [text_generation_pipeline](https://ml.azure.com/registries/azureml/components/text_generation_pipeline)
```

Comment:
Shall we create a table also add links to the respective task notebooks as a 2nd column?